pkgs/development/python-modules/typecode/libmagic.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pkgs
+}:
+buildPythonPackage rec {
+  pname = "typecode-libmagic";
+  version = "21.4.4";
+
+  src = fetchFromGitHub {
+    owner = "nexB";
+    repo = "scancode-plugins";
+    rev = "v${version}";
+    sha256 = "xnUGDMS34iMVMGo/nZwRarGzzbj3X4Rt+YHvvKpmy6A=";
+  };
+
+  postPatch = ''
+    cd builtins/typecode_libmagic-linux
+  '';
+
+  preBuild = ''
+    pushd src/typecode_libmagic
+
+    cp ${pkgs.file}/share/misc/magic.mgc data/magic.mgc
+    cp ${pkgs.file}/lib/libmagic.so lib/libmagic.so
+    cp ${pkgs.zlib}/lib/libz.so lib/libz-lm539.so.1
```

```suggestion
    cp ${file}/share/misc/magic.mgc data/magic.mgc
    cp ${file}/lib/libmagic.so lib/libmagic.so
    cp ${zlib}/lib/libz.so lib/libz-lm539.so.1
```

I think we should use ``ln -s`` here.